Tools/GNUMake/sites/Make.nersc

```diff
@@ -24,6 +24,12 @@ ifeq ($(which_computer),$(filter $(which_computer),perlmutter))
   endif
   endif
 
+  ifeq (($USE_CUDA),TRUE)
+    includes += -Xcompiler="-march=znver3" 
+  else
+    includes += -march=znver3
```

Comment:
This might be gnu only.

Reply:
Very true. I can add in that ifeq.

Not sure what to do about Nvidia or the upcoming LLVM, though, other than a nested if statement.